### ISSUE TITLE
Check if `class_weight` parameter is to blame for poor performance of minority class

### ISSUE THREAD

**Issue by admivsn** [admivsn/ml-at-risk#1]
For classes with normalised counts: `{0: 0.75, 1: 0.25}` it seems like the class weight parameter should be the inverse (i.e. `{0: 0.25, 1: 0.75}` or `{0: 1, 1: 3}`) to penalise the majority class. Alternatively, there is an `sklearn` function to compute this parameter:

https://scikit-learn.org/stable/modules/generated/sklearn.utils.class_weight.compute_class_weight.html
